.github/component_owners.yml

```
@@ -17,67 +21,101 @@ components:
+  plugins/node/instrumentation-socket.io:
+    - mottibec
+    - rauno56
   plugins/node/instrumentation-tedious:
     - rauno56
   plugins/node/opentelemetry-instrumentation-aws-lambda:
     - carolabadeer
   plugins/node/opentelemetry-instrumentation-aws-sdk:
     - carolabadeer
     - blumamir
+  plugins/node/opentelemetry-instrumentation-bunyan: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-cassandra: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-connect: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-dns: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-express: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-fastify: []
+    # Unmaintained?
   plugins/node/opentelemetry-instrumentation-generic-pool:
     - rauno56
   plugins/node/opentelemetry-instrumentation-graphql:
     - obecny
+  plugins/node/opentelemetry-instrumentation-hapi: []
+    # Unmaintained?
   plugins/node/opentelemetry-instrumentation-ioredis:
     - blumamir
     - naseemkullah
   plugins/node/opentelemetry-instrumentation-knex:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-lru-memoizer:
-    - blumamir
+  plugins/node/opentelemetry-instrumentation-koa: []
+    # Unmaintained?
   plugins/node/opentelemetry-instrumentation-memcached:
     - rauno56
   plugins/node/opentelemetry-instrumentation-mongodb:
     - osherv
-  plugins/node/opentelemetry-instrumentation-mongoose:
-    - blumamir
   plugins/node/opentelemetry-instrumentation-mysql:
     - haddasbronfman
+  plugins/node/opentelemetry-instrumentation-mysql2: []
+    # Unmaintained?
   plugins/node/opentelemetry-instrumentation-nestjs-core:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-redis:
-    - blumamir
+  plugins/node/opentelemetry-instrumentation-net: []
+    # Unmaintained?
+  plugins/node/opentelemetry-instrumentation-pg:
+    - rauno56
+  plugins/node/opentelemetry-instrumentation-pino: []
+    # Unmaintained?
   plugins/node/opentelemetry-instrumentation-redis-4:
     - blumamir
+  plugins/node/opentelemetry-instrumentation-redis:
+    - blumamir
   plugins/node/opentelemetry-instrumentation-restify:
     - rauno56
   plugins/node/opentelemetry-instrumentation-router:
     - rauno56
-  plugins/node/opentelemetry-instrumentation-pg:
-    - rauno56
-  plugins/node/opentelemetry-instrumentation-socket.io:
-    - mottibec
-    - rauno56
+  plugins/node/opentelemetry-instrumentation-winston: []
+    # Unmaintained?
   plugins/web/opentelemetry-instrumentation-document-load:
     - obecny
   plugins/web/opentelemetry-instrumentation-long-task:
     - mhennoch
     - t2t2
   plugins/web/opentelemetry-instrumentation-user-interaction:
     - obecny
+  plugins/web/opentelemetry-plugin-react-load: []
+    # Unmaintained?
   propagators/opentelemetry-propagator-aws-xray:
     - carolabadeer
+  propagators/opentelemetry-propagator-grpc-census-binary: []
+    # Unmaintained?
```

Originally contributed by @nkelly75  

Would you care to take ownership?

```suggestion
  propagators/opentelemetry-propagator-grpc-census-binary: []
    - nkelly75
```